main.html

```diff
@@ -144,5 +144,22 @@ <h3 class="worktitle">My Recent Work</h3>
             </div>
         </div>
     </section>
+    <section class="contact-form">
+        <h2 class="contactUs">CONTACT US</h2>
+        <p class="contactUs-message">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/mjvlqywj" class="name" method="post">
+            <input type="text" class="name" placeholder="Full Name" name="fullName" minlength="1" maxlength="30" required>
+            <input type="email" class="contactemail" placeholder="Email" name="email" required>
+            <textarea name="request" id="request1" cols="30" rows="10" maxlength="500" placeholder="Enter Text here"></textarea>
+        <button type="submit" class="submit">Submit Request</button>
+        </form>
+        <img class="separator" src="./pictures/vector4.png" alt="line to separate information">
+        <div class="findMe">
+            <a href="https://www.linkedin.com/"><img class="link" src="pictures/TimeAndIcons/linkedin.png" alt=""></a>
+            <a href="https://github.com/Flunavgt"><img class="link" src="pictures/TimeAndIcons/github.png" alt=""></a>
+            <a href="https://twitter.com/?lang=en"><img class="link" src="pictures/TimeAndIcons/twitter.png" alt=""></a>
+        </div>
```

Comment:
- [ ] like the previous reviewer noted, kindly add your social icons so they look like the figma version. they will serve as links to your online sites so they are useful. 
your version 
![Screenshot (210)_LI](https://user-images.githubusercontent.com/65298048/177434495-b2d55d0b-79a7-4cbc-801a-b36cbfa412f4.jpg)
figma version 
![Screenshot (209)_LI](https://user-images.githubusercontent.com/65298048/177434563-ccadde88-d246-4a14-8248-cee3df5a9940.jpg)